resources/nats-streaming/profile-evaluation.yaml

```diff
@@ -0,0 +1,5 @@
+global:
```

Comment:
Is it possible not to use `global`? It breaks the consistency with other profile-evaluation files.

Reply:
Values.yaml uses global. I believe I have to use the same format to be able to overwrite.